src/GraphQLParser/Visitors/SDLWriter.cs

```diff
@@ -854,7 +902,7 @@ private int GetLevel(TContext context)
                         ++level;
                 }
 
-                if (currentNode is GraphQLDescription)
+                if (currentNode is GraphQLDescription || currentNode is GraphQLDirectiveDefinition)
```

Comment:
I still do not know how to do it better with indentation (and new lines). I'll think about that later after all PRs around grammar.